TESTING.md

```diff
@@ -14,6 +14,32 @@ Before you can run tests locally you must have:
 
         $ curl https://sdk.cloud.google.com | bash
 
+## System Dependencies
+
+Running the complete suite may require system dependencies. Most of these should be installable via typical package managers like `apt-get` and [brew](http://brew.sh/).
+
+For example, to install portaudio:
+
+    brew install portaudio
+
+If you have trouble installing a necessary system library, try filing an issue.
+
+### Troubleshooting
+
+#### PortAudio on OS X
+
+If you see the error
+    
+    fatal error: 'portaudio.h' file not found
+
+Try adding the following to your ~/.pydistutils.cfg file,
+substituting in your appropriate brew Cellar directory:
+
+    include_dirs=/usr/local/Cellar/portaudio/19.20140130/include/
+    library_dirs=/usr/local/YourUsername/homebrew/Cellar/portaudio/19.20140130/lib/
```

Comment:
Maybe, what do you have in mind in terms of linking?

The issue is right now if someone wants to just run the whole test suite, we should make it fairly straightforward to do so. So it's not speech specific in that sense. But, we probably also want instructions in the speech samples and don't want to repeat ourselves.